cash_register.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+
+# This is the CashRegister Class
+class CashRegister
+  def initialize
+    @total = 0.00
+    two_decimal = format('%.2f', @total)
+    puts "The cash register balance is $#{two_decimal}"
+  end
+
+  def purchase(item)
+    food_items = ['Organic Bears Milk', 'Wagyu Ribeye', 'Smoked Bacon Salt']
+    food = food_items.sample(1).join
+    @total += item
+    two_decimal = format('%.2f', @total)
+    puts "You just purchased the #{food} for $#{format('%.2f', item)}. \b
+What a deal! The balance is $#{two_decimal}."
+  end
+
+  def pay(amount)
+    @total -= amount
+
+    if @total >= 0
+      two_decimal = format('%.2f', @total)
+      puts "You payed $#{amount}. Your balance is $#{two_decimal}."
+    elsif @total < 0
+      puts "You payed $#{amount}. Here's $#{format('%.2f', @total * -1)}."
+      puts "0 items in cart. The balance is $#{format('%.2f', 0.00)}."
+    else
+      puts 'Error message!'
```

Comment:
This would never occur.  When would a number be both not >= 0 (line 23) and not < 0 (line 26).  I'd change line 26 to an `else`.